src/js/views/main/index.js

```diff
@@ -131,49 +76,70 @@ class MainView extends Component {
       });
    };
 
-   componentDidMount() {
-      this.viewData();
-   }
+   handleDrop = file => {
+      this.setState(state => {
+         state.fileStack.push(file[0]);
+         return state;
+      });
+   };
+
+   deleteFile = index => {
+      this.setState(state => {
+         state.fileStack.splice(index, 1);
+         return state;
+      });
+   };
+
+   sendFiles = () => {
+      this.setState({
+         uploading: true
+      });
+      setTimeout(() => {
+         this.setState({
+            showLoader: true
+         });
+         const url =
+            "http://ec2-54-86-77-144.compute-1.amazonaws.com:5000/test/test";
```

Comment:
Will probably be used more than once. Consider making URL global or moving file system handling to own file